frontend/www/js/omegaup/components/course/AssignmentList.vue

```diff
@@ -203,59 +142,32 @@ library.add(fas);
   },
 })
 export default class CourseAssignmentList extends Vue {
-  @Prop() assignments!: omegaup.Assignment[];
+  @Prop() content!: types.CourseAssignment[];
   @Prop() courseAlias!: string;
   @Prop() assignmentFormMode!: omegaup.AssignmentFormMode;
 
-  testsOrderChanged = false;
-  homeworksOrderChanged = false;
+  contentOrderChanged = false;
   T = T;
   AssignmentFormMode = omegaup.AssignmentFormMode;
 
-  get homeworks(): omegaup.Assignment[] {
-    return this.assignments.filter((assignment: omegaup.Assignment) => {
-      return assignment.assignment_type == 'homework';
-    });
-  }
-
-  get tests(): omegaup.Assignment[] {
-    return this.assignments.filter((assignment: omegaup.Assignment) => {
-      return assignment.assignment_type == 'test';
-    });
-  }
-
   assignmentUrl(assignment: omegaup.Assignment): string {
     return `/course/${this.courseAlias}/assignment/${assignment.alias}/`;
   }
 
-  sortHomeworks(event: any): void {
-    this.homeworks.splice(
-      event.newIndex,
-      0,
-      this.homeworks.splice(event.oldIndex, 1)[0],
-    );
-    this.homeworksOrderChanged = true;
-  }
-
-  sortTests(event: any): void {
-    this.tests.splice(
+  sortContent(event: any): void {
+    this.content.splice(
       event.newIndex,
       0,
-      this.tests.splice(event.oldIndex, 1)[0],
+      this.content.splice(event.oldIndex, 1)[0],
     );
-    this.testsOrderChanged = true;
+    this.contentOrderChanged = true;
   }
 
-  saveNewOrder(type: string): void {
+  saveNewOrder(): void {
     let param: string[] = [];
-    if (type === 'homeworks') {
-      param = this.homeworks.map((homework) => homework.alias);
-      this.homeworksOrderChanged = false;
-    } else {
-      param = this.tests.map((test) => test.alias);
-      this.testsOrderChanged = false;
-    }
-    this.$emit(`emit-sort-${type}`, this.courseAlias, param);
+    param = this.content.map((assignment) => assignment.alias);
+    this.contentOrderChanged = false;
+    this.$emit('emit-sort-content', this.courseAlias, param);
```

Comment:
```suggestion
    this.contentOrderChanged = false;
    this.$emit('emit-sort-content', this.courseAlias, this.content.map((assignment) => assignment.alias));
```

ya no se necesita declarar `param`

Reply:
Cierto, se quita.